suite/smoke/keeper_test.go

```diff
@@ -179,7 +172,7 @@ var _ = Describe("Keeper suite @keeper", func() {
 			networks.Default.GasStats().PrintStats()
 		})
 		By("Tearing down the environment", func() {
-			err = actions.TeardownSuite(env, networks, utils.ProjectRoot, nil)
+			err = actions.TeardownSuite(env, networks, utils.ProjectRoot, chainlinkNodes, nil)
```

Comment:
That will break all the stuff in Terra/Solana on update, can we start to use functional options there?

Reply:
Messaging you